src/js/common/utils/accessible-menus.js

```diff
@@ -371,7 +372,10 @@ export default function addMenuListeners(menuElement, closeOnResize = false) {
     const inMenu = targetLi.parentElement.getAttribute('role').toLowerCase() === 'menu';
 
     // Handle opening (and closing) menus
-    if (inMenubar) {
+    if (e.target.classList.contains('back-button')) {
```

Comment:
I wasn't sure if this should be handled in here or `mega-menu.js` since all other logic for small screen menus is in `mega-menu`. In the end, I decided to put it here for a couple reasons:
1. The helper functions were already here; why recreate the same functionality in `mega-menu.js`?
1. Other menus might have a back button to navigate back to a previous menu level

If anybody has strong opinions about this, I'd be happy to hear them.